doc/Install.md

```diff
@@ -97,6 +97,12 @@ Registration errors should all be recoverable automatically.
 If you get any *critical* failure at this point, it generally indicates the database was not installed correctly.
 You might wish to move/rename .htconfig.php to another name and empty (called 'dropping') the database tables, so that you can start fresh.
 
+#### Automatic Installation
+
+In case you want to automatically install friendica, you have to create a `autoconfig.php` file inside the `config/` directory.
```

Comment:
Isn't the automatic install supposed to work alongside Docker? Without any environment tinkering, the automatic install will fail, right?

Reply:
Yes, because than you have to manually set the values in `autoconfig.php`
theoretically you can set any value you prefer and the auto-installer would work.

Shall I change the variables to fixed values to suggest a more general usage?

Reply:
That would be great indeed.

Reply:
I added a doc
https://github.com/nupplaphil/friendica/blob/1ff54c653e442ebc47b29fc38b5cf14295bc7d0e/doc/Docker.md

Please can you check if there are some spelling mistakes. My english grammar isn't the best ^^